controllers/operands/cdi_test.go

```diff
@@ -653,15 +659,18 @@ var _ = Describe("CDI Operand", func() {
 
 				Expect(foundCDI.Spec.Config).ToNot(BeNil())
 				Expect(foundCDI.Spec.Config.InsecureRegistries).To(BeNil())
+				Expect(foundCDI.Spec.Config.DataVolumeTTLSeconds).To(BeNil())
 			})
 
-			It("should modify InsecureRegistries according to HCO CR", func() {
+			It("should modify CDIConfig according to HCO CR", func() {
 				existingCDI, err := NewCDI(hco)
 				Expect(err).ToNot(HaveOccurred())
 				existingCDI.Spec.Config.InsecureRegistries = []string{"first:5000", "second:5000", "third:5000"}
+				existingCDI.Spec.Config.DataVolumeTTLSeconds = &[]int32{2}[0]
```

Comment:
Please use `pointer.Int32(2)` instead:
```suggestion
				existingCDI.Spec.Config.DataVolumeTTLSeconds = pointer.Int32(2)
```